### DISCUSSION TITLE
[FEATURE] Insert a py-repl with multi-line code inside (using ideally JS)

### DISCUSSION THREAD

**Discussion by ckavidas** [pyscript/pyscript#1176] [Q&A]
**Is your feature request related to a problem? Please describe.**
I would like to be able to insert a py-repl element with multi-line innertext values from either javascript or from pyscript (I am a total JS newbie)

**Describe the solution you'd like**
A clear example inserting a py-repl element with multi-line innertext values

**Describe alternatives you've considered**
- I have tried using backticked strings but it inserts <br> tags whenever a new line is detected. 
- I also tried using "\n" and "\t" which didn't work for me either.
- There is also the whitespace issue when trying to generate multi-line code.

**Additional context**
I am building a "jupyter notebook on the go" application using the py-repl tags. I have a javascript function that produces "cells" and adds buttons to add/remove cells.

The next step for me is to add some code snippets
- A "upload file" code snippet which creates a py-repl with the code needed to create an upload file form
- A "download file" code snippet which creates a py-repl with the code needed to download a file (requires a user to update some values)
- A "Install package" code snippet which creates a py-repl with the code needed to install packages using 

The idea is to press a button and create those common code snippets while performing whatever operation is necessary. (Also serves as a good test-page for pyscript functionality I guess)

**Comment by marimeireles** [comment]
Oh, I'd be interested in learning more about your progress in this.
I think the space issue is tough to tackle as it's related to the very way HTML is implemented.

**Comment by chenchen100** [comment]
use `<py-repl src='py_code.py'></py-repl>`
